spec/components/layout_super_navigation_header_spec.rb

```diff
@@ -166,13 +166,11 @@ def component_name
     render_component({ ga4_tracking: true })
 
     assert_select "header[data-module='gem-track-click ga4-event-tracker ga4-link-tracker']"
-    assert_select "a[data-ga4-link]", count: 29
+    assert_select "a[data-ga4-link]", count: 23
     assert_select 'a[data-ga4-link=\'{"event_name":"navigation","type":"header menu bar","external":"false","text":"GOV.UK","section":"Logo","index_link":1,"index_section":0,"index_section_count":2,"index_total":1}\']'
     assert_select 'a[data-ga4-link=\'{"event_name":"navigation","type":"header menu bar","index_section":1,"index_link":1,"index_section_count":4,"index_total":16,"section":"Services and information"}\']'
     assert_select 'a[data-ga4-link=\'{"event_name":"navigation","type":"header menu bar","index_section":1,"index_link":16,"index_section_count":4,"index_total":16,"section":"Services and information"}\']'
     assert_select 'a[data-ga4-link=\'{"event_name":"navigation","type":"header menu bar","index_section":2,"index_link":1,"index_section_count":4,"index_total":6,"section":"Government activity"}\']'
     assert_select 'a[data-ga4-link=\'{"event_name":"navigation","type":"header menu bar","index_section":2,"index_link":6,"index_section_count":4,"index_total":6,"section":"Government activity"}\']'
```

Comment:
@hannalaakso I think I may not have been clear enough on the above - `index_section_count` for all of the links here should change from 4 to 3, because the number of sections has reduced.

Indexes are pretty confusing - I'm currently writing some docs to try to clarify this https://github.com/alphagov/govuk_publishing_components/pull/3919

Reply:
> @hannalaakso I think I may not have been clear enough on the above - `index_section_count` for all of the links here should change from 4 to 3, because the number of sections has reduced.
> 
> Indexes are pretty confusing - I'm currently writing some docs to try to clarify this #3919

Hello @andysellick - is there a further change needed here? I have moved onto completing this today and have just merged this as it is. The change seemed to have been made but maybe that's not quite the case yet ...

Reply:
No worries, I've gone ahead and made the change here: https://github.com/alphagov/govuk_publishing_components/pull/3923

If we can get this included in the new release, that'd be great, thanks!